src/actions/CryptoExchangeActions.js

```diff
@@ -289,36 +289,40 @@ export const shiftCryptoCurrency = (swapInfo: GuiSwapInfo) => async (dispatch: D
 
     let category: string
     let name: string
+    let notes: string
     if (pluginId === 'transfer') {
       category = sprintf('transfer:%s %s %s', fromCurrencyCode, s.strings.word_to_in_convert_from_to_string, toWallet.name)
       name = toWallet.name || ''
+
+      const recipientAddress = (await toWallet.getReceiveAddress()).publicAddress
+      notes = sprintf(s.strings.transfer_notes_metadata_generic, recipientAddress)
```

Comment:
Use existing tx_notes_metadata_recipient_address string and concatenate the recipient address